supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -166,6 +164,27 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     MetadataJobKey(jobDescriptorKey.call.fullyQualifiedName, jobDescriptorKey.index, jobDescriptorKey.attempt)
   }
 
+  override def receive: Receive = LoggingReceive {
+    case AbortJobCommand =>
+      serviceRegistryActor ! GetMetadataQueryAction(MetadataQuery.forKey(metadataKey("jobId")))
+    case MetadataLookupResponse(query, events) if query.key.contains("jobId") =>
+      /** Currently this actor can only get a MetadataLookupResponse when requesting a
+        * job ID for abort.  If this actor becomes more complex in the future, then this
+        * should probably be received in an FSM state.
```

Comment:
Yes, partially, though I just added this comment so it was clear why this actor was receiving a Job ID from metadata... though I guess the code below is self-explanatory enough... next person to modify this code can remove the comment if they want :)